pkgs/development/python-modules/sane/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, saneBackends
+}:
+
+buildPythonPackage rec {
+  pname = "sane";
+  version = "2.8.2";
+
+  src = fetchPypi {
+    inherit version;
+    pname = "python-sane";
+    sha256 = "0sri01h9sld6w7vgfhwp29n5w19g6idz01ba2giwnkd99k1y2iqg";
+  };
+
+  buildInputs = [
+    saneBackends
+  ];
+
+  meta = with lib; {
+    homepage = "https://github.com/python-pillow/Sane";
+    description = "Python interface to the SANE scanner and frame grabber ";
+    license = "https://github.com/python-pillow/Sane/blob/master/COPYING";
```

Comment:
this is MIT-like, but some of the wording is slightly different.
but a typical license usually has many attributes:
```
nix-repl> lib.licenses.mit
{ fullName = "MIT License"; shortName = "mit"; spdxId = "MIT"; url = "http://spdx.org/licenses/MIT.html"; }
```

Reply:
Great, didn't know what to put there.